src/App.tsx

```diff
@@ -1,9 +1,26 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
+const filter = (str: string, searchStr: string) => {
```

Comment:
Consider renaming the 'filter' function to a more descriptive name, such as 'filterMoviesByQuery'.